feconf.py

```diff
@@ -608,7 +608,8 @@ def get_empty_ratings():
 ])
 
 # The header for the "Featured Activities" category in the library index page.
-LIBRARY_CATEGORY_FEATURED_ACTIVITIES = 'Featured Activities'
+LIBRARY_GROUP_FEATURED_ACTIVITIES_I18N_ID = \
```

Comment:
Don't use backslashes at ends of lines. See [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide).